staging/src/k8s.io/client-go/util/workqueue/metrics.go

```diff
@@ -103,6 +114,23 @@ func (m *defaultQueueMetrics) done(item t) {
 	}
 }
 
+func (m *defaultQueueMetrics) updateUnfinishedWork() {
+	var total float64
+	if m.processingStartTimes != nil {
+		for _, t := range m.processingStartTimes {
+			total += sinceInMicroseconds(t)
```

Comment:
Ah, so this is (items * time).  This is kind of a confusing unit.  Don't we really just care about the oldest thing in the queue?

Reply:
I could report that, but you wouldn't be able to tell the difference between one thing stuck and two things stuck.

Reply:
I agree that the unit is a little confusing, but emitting this as raw data is probably okay. Whatever thing is responsible for scraping this data can post-process before turning it into an actual (and more useful and less confusing) metric datapoint.

Reply:
It is a confusing unit, but I think all the things that we can do to make it less confusing are lossy (or require making assumptions about the duration of the things that are being done) and therefore best done in the monitoring system.

Reply:
Github stuck my response in the wrong place:

> If you plot this over time, the slope will tell you the difference between those scenarios.

>I'm open to reporting a distribution, of course, but I don't have a single thing to observe and it wasn't clear how to get around that.

Reply:
I think I might change the unit to be seconds, though. Microseconds is far more granular than necessary.

Reply:
Unit changed, and I expanded the help string for the metric since it wasn't obvious how to use it.